packages/aws-rfdk/lib/deadline/scripts/python/test_configure_identity_registration_settings.py

```diff
@@ -0,0 +1,317 @@
+        ]
+
+    def test_create(self):
+        connection_subnet_ids = [
+            "subnet-aaaaaaa" # Used by REG_SETTING_RFDK_1
+        ]
+        source_subnets = [
+            subject.SourceSubnet(subnet_id="subnet-2222222", role="Client", registration_status="Registered"), # Used by REG_SETTING_RFDK_2,
+        ]
+        subnet_to_cidr = {
+            "subnet-aaaaaaa": "111.111.0.0/16", # "111.111.*.*" Used by REG_SETTING_RFDK_1
+            "subnet-2222222": "3.3.0.0/18", # "3.3.0-63.*" # Used by REG_SETTING_RFDK_2
+        }
+
+        subject.create_and_update_settings(self.dl_secrets, self.prior_rfdk_settings, connection_subnet_ids, source_subnets, subnet_to_cidr)
+
+        expected = subject.LoadBalancerIdentityRegistrationSetting.from_json({
+            "ConnectionIpFilterType":"IPv4Match",
+            "ConnectionIpFilterValue":"111.111.*.*", # Similar to REG_SETTING_1
+            "SourceIpFilterType":"IPv4Match",
+            "SourceIpFilterValue":"3.3.0-63.*", # Similar to REG_SETTING_2
+            "SettingsId":"",
+            "SettingsName":"RfdkSubnet|subnet-aaaaaaa|subnet-2222222",
+            "IsEnabled":True,
+            "DefaultStatus":"Registered",
+            "DefaultRole":"Client"
+        })
+
+        self.dl_secrets.run_str.assert_called_once_with(
+            "CreateLoadBalancerIdentityRegistrationSetting",
+            expected.settings_name,
+            expected.connection_ip_filter_type,
+            expected.connection_ip_filter_value,
+            expected.source_ip_filter_type,
+            expected.source_ip_filter_value,
+            expected.default_role,
+            expected.default_status,
+            str(expected.is_enabled),
+        )
+
+    def test_update(self):
+        connection_subnet_ids = [
+            "subnet-aaaaaaa" # Used by REG_SETTING_RFDK_1
+        ]
+        source_subnets = [
+            subject.SourceSubnet(subnet_id="subnet-1111111", role="Server", registration_status="Pending"), # ID ised by REG_SETTING_RFDK_1,
+        ]
+        subnet_to_cidr = {
+            "subnet-aaaaaaa": "111.111.0.0/16", # "111.111.*.*" Used by REG_SETTING_RFDK_1
+            "subnet-1111111": "123.123.123.0/24", # "123.123.123.*" # Used by REG_SETTING_RFDK_1
+        }
+
+        subject.create_and_update_settings(self.dl_secrets, self.prior_rfdk_settings, connection_subnet_ids, source_subnets, subnet_to_cidr)
+
+        expected = subject.LoadBalancerIdentityRegistrationSetting.from_json({
+            "ConnectionIpFilterType":"IPv4Match",
+            "ConnectionIpFilterValue":"111.111.*.*",
+            "SourceIpFilterType":"IPv4Match",
+            "SourceIpFilterValue":"123.123.123.*",
+            "SettingsId":"settings1",
+            "SettingsName":"RfdkSubnet|subnet-aaaaaaa|subnet-1111111",
+            "IsEnabled":True,
+            "DefaultStatus":"Pending",
+            "DefaultRole":"Server"
+        })
```

Comment:
What do you think about sourcing the values for some of these expected fields from literals to using attributes of `REG_SETTINGS_RFDK_1` where applicable? just to help depict what fields we expect to be updated vs what we expect to remain unchanged?